resources/views/bank/nordigen/connect.blade.php

```diff
@@ -0,0 +1,67 @@
+@extends('layouts.ninja')
+@section('meta_title', ctrans('texts.new_bank_account'))
+
+@push('head')
+
+<link href="https://unpkg.com/nordigen-bank-ui@1.5.2/package/src/selector.min.css" rel="stylesheet" />
+
+@endpush
+
+@section('body')
+
+<div id="institution-content-wrapper"></div>
+
+@endsection
+
+@push('footer')
+
+<script type='text/javascript' src='https://unpkg.com/nordigen-bank-ui@1.5.2/package/src/selector.min.js'></script>
+
+<script>
+
+
+    // Pass your redirect link after user has been authorized in institution
+    const config = {
+        // Text that will be displayed on the left side under the logo. Text is limited to 100 characters, and rest will be truncated.
+        text: "Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean mavdvd",
+        // Logo URL that will be shown below the modal form.
+        logoUrl: "{{ $account && !$account->isPaid() ? asset('images/invoiceninja-black-logo-2.png') : (isset($company) && !is_null($company) ? $company->present()->logo() : '') }}",
+        // Will display country list with corresponding institutions. When `countryFilter` is set to `false`, only list of institutions will be shown.
+        countryFilter: false,
+        // style configs
+        styles: {
+            // Primary
+            // Link to google font
+            fontFamily: 'https://fonts.googleapis.com/css2?family=Roboto&display=swap', // @todo replace to match german law: not use google fonts and use local instead
```

Comment:
@turbo124 does in uses google fonts? are there local hosted versions of it?

Reply:
/public/fonts/

we do store roboto there, I believe some selfhost users much prefer to load the font locally rather than ping a cdn